### ISSUE TITLE
Replace empty tables with informative text

### ISSUE THREAD

**Issue by rdeltour** [daisy/ace#61]
In the accessibility metadata summary for instance, when there is no such metadata, we currently display an empty table. That can be confusing. The empty table should be replaced by informative text in this case.
